torch/nn/quantized/modules/normalization.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, normalized_shape, weight, bias, scale, zero_point, eps=1e-5,
         self.zero_point = zero_point
 
     def forward(self, input):
-        return torch.ops.quantized.layer_norm(
+        return torch.quantized_layer_norm(
```

Comment:
Wouldn't having these methods in the quantized name-space be preferrable?

Reply:
please see the comment above - the reasoning is the same here